mbedtls/src/lib.rs

```diff
@@ -8,6 +8,7 @@
 
 #![deny(warnings)]
 #![allow(unused_doc_comments)]
+#![allow(suspicious_auto_trait_impls)]
```

Comment:
What caused the need for this & is there a way we can avoid this?

Reply:
It's the `unsafe impl Send for MbedtlsList<Certificate>` (`Certificate` is a real type). I believe it's fine as long as we're not using `MbedtlsList<AnythingElse>` (which I don't believe we actually do).

The real fix is likely `impl Send for MbedtlsList<T> where T: Send` or something along those lines.